dev/integration_tests/flutter_gallery/test_driver/e2e_util.dart

```diff
@@ -0,0 +1,159 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The following is copied from dev/benchmarks/macrobenchmarks/test/util.dart
+// TODO(CareF): Remove this file when the following lands to the correct place
+
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flutter/scheduler.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:e2e/e2e.dart';
+
+/// The maximum amount of time considered safe to spend for a frame's build
+/// phase. Anything past that is in the danger of missing the frame as 60FPS.
+///
+/// Changing this doesn't re-evaluate existing summary.
+Duration kBuildBudget = const Duration(milliseconds: 16);
+
+Future<void> watchPerformance(
```

Comment:
I think our conclusion is both this and the summarizer can first go to a separate package in a personal repo, and then to flutter/packages.

Reply:
Oh, sorry that I remembered wrong. I had a second thought that `watchPerformance` might go to E2E because it seems to just for reporting with the E2E binding. Either way should work and the difference seems to be either letting E2E depend on your new package, or letting your new package depend on E2E.

Reply:
Done.